mqtt-cs/pom.xml

```diff
@@ -70,6 +74,12 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.rocketmq</groupId>
+            <artifactId>meta</artifactId>
```

Comment:
where is this packeage.

Reply:
meta in master branch. It has been removed in develop branch